app/src/main/java/ch/epfl/sdp/artificial_intelligence/RandomEnemyGenerator.java

```diff
@@ -10,6 +10,11 @@
 import ch.epfl.sdp.geometry.Area;
 import ch.epfl.sdp.geometry.GeoPoint;
 
+
+/**
+ * Class that randomly generates an enemy
+ * This class differs from RandomGenerator, as it specifically generates an enemy (and nothing else)
+ */
 public class RandomEnemyGenerator extends EnemyGenerator {
```

Comment:
I was wondering is it really necessary to have abstract class EnemyGenerator, because this one is only extended by RandomEnemyGenerator

Reply:
Indeed, it is not very useful. I will make the changes now and delete it.